Dockerfile

```diff
@@ -1,52 +1,51 @@
 #
-# Spiderfoot Dockerfile 
+# Spiderfoot Dockerfile
 #
 # http://www.spiderfoot.net
 #
-# Written by: Michael Pellon <m@pellon.io>
+# Written originally by: Michael Pellon <m@pellon.io>
+# Further optimized by António Meireles <antonio.meireles@reformi.st>
 #
 # Usage:
 #
 #   sudo docker build -t spiderfoot .
 #   sudo docker run -it -p 8080:8080 spiderfoot
 
 # Pull the base image.
-FROM ubuntu:14.04
+FROM ubuntu:latest
+
+ENV VERSION 2.6.1
+ENV RELEASE ${VERSION}-final
+ENV TARBALL https://github.com/smicallef/spiderfoot/archive/v${RELEASE}.tar.gz
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM linux
+
+EXPOSE 8080
 
 # Install pre-requisites.
-RUN apt-get update && apt-get install -y \
-  build-essential \
-  curl \
-  git \ 
-  libssl-dev \
-  libxml2-dev \
-  libxslt1-dev \
-  python-pip  \
-  python-dev \
-  python-lxml \
-  swig \
-  --no-install-recommends \
-  && rm -rf /var/lib/apt/lists/* \
-  && pip install cherrypy lxml mako M2Crypto netaddr
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        curl git python-pip python-lxml python-m2crypto python-netaddr \
```

Comment:
Not a good thing to have multiple installs per line. https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/